Source/gui/SNTBinaryMessageWindowView.swift

```diff
@@ -184,29 +206,6 @@ struct SNTBinaryMessageEventView: View {
       Divider()
```

Comment:
Can you delete the comment on lines 187-191

Reply:
Done